scripts/deploy_xcframework.sh

```diff
@@ -36,6 +36,8 @@ fi
 PROD_BRANCH="production-$(date +"%Y%m%d-%s")"
 git checkout -b $PROD_BRANCH
 
+./scripts/update_version.sh ${1}
```

Comment:
do we need to create a new commit before cutting the branch?

Reply:
wouldn't the changes be added in line 61?

Reply:
ahh, yes. they would. it won't go to the master branch, but will be pushed to the prod branch. that's probably fine.

Reply:
would be it be better practice to push it to master?